pkg/minikube/bootstrapper/bsutil/ktmpl/v1beta2.go

```diff
@@ -41,9 +41,12 @@ nodeRegistration:
 apiVersion: kubeadm.k8s.io/v1beta2
 kind: ClusterConfiguration
 {{ if .ImageRepository}}imageRepository: {{.ImageRepository}}
-{{end}}{{range .ExtraArgs}}{{.Component}}:
+{{end}}{{range .ComponentOptions}}{{.Component}}:
+{{- range $k, $v := .Pairs }}
+  {{$k}}: {{$v}}
+{{- end}}
   extraArgs:
-{{- range $i, $val := printMapInOrder .Options ": " }}
+{{- range $i, $val := printMapInOrder .ExtraArgs ": " }}
```

Comment:
if there is no extra args at all ? would this block error ?

Reply:
This block will just proceed as if nothing was there.